src/cool_seq_tool/mappers/exon_genomic_coords.py

```diff
@@ -49,6 +39,185 @@ class ExonCoord(BaseModelForbidExtra):
     )
     alt_strand: Strand = Field(..., description="Strand.")
 
+    model_config = ConfigDict(
+        json_schema_extra={
+            "example": {
+                "ord": 0,
+                "tx_start_i": 0,
+                "tx_end_i": 234,
+                "alt_start_i": 154191901,
+                "alt_end_i": 154192135,
+                "alt_strand": Strand.NEGATIVE,
+            }
+        }
+    )
+
+
+class _TxSegment(BaseModelForbidExtra):
```

Comment:
I actually think that we should be declaring these without a leading underscore because they're a subcomponent of the return object of core public methods. Like, if I decide to change the `exon_ord` property to `exon_i` during a free day, that's something that should be marked as a breaking change since it affects the structure of the outputof `tx_segment_to_genomic()`

Reply:
Works for me